src/uct/base/uct_md.h

```diff
@@ -32,34 +32,59 @@ struct uct_md_config {
 };
 
 
-/**
- * Memory domain operations
- */
-struct uct_md_ops {
-    void         (*close)(uct_md_h md);
+    typedef void (*uct_md_close_func_t)(uct_md_h md);
```

Comment:
why do we have indentation here?

Reply:
wanted to put all typedefs into separate structure to eliminate prefix in datatypes... doesn't work :)

indentation is fixed